homeassistant/components/anova_sous_vide/config_flow.py

```diff
@@ -0,0 +1,38 @@
+"""Config flow for Anova."""
+from __future__ import annotations
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
```suggestion
```

There is no logging used in the config flow. So there is no actual need to define it as well as import logging